lib/screens/app/ecosystem/overview.dart

```diff
@@ -56,6 +58,17 @@ class _OverviewState extends State<Overview> {
     NavigationService.of(context).navigateTo(Routes.plantSeeds);
   }
 
+  void onDonate() {
+    NavigationService.of(context).navigateTo(
+      Routes.transferForm,
+      TransferFormArguments(
+        "balisupport.org",
+        "balisupports",
+        "https://balisupport.org/wp-content/uploads/2020/04/Asset-12@3x-1-300x296.png",
```

Comment:
this hard coded URL will eventually be an issue.

Reply:
Donation has nothing to do with the token change?!

Reply:
Needs to be configurable by firebase config